src/main/java/Duke.java

```diff
@@ -17,23 +17,40 @@ public static void main(String[] args) {
         Scanner sc = new Scanner(System.in);
         String userInput;
         String output;
-        List<String> tasks = new ArrayList<>();
+        List<task> tasks = new ArrayList<>();
         do{
             userInput = sc.nextLine();
+
+            int taskIndex = 0;
+            if(userInput.contains("done")){
+                String[] split_ = userInput.split(" ");
+                userInput = split_[0];
+                taskIndex = Integer.valueOf(split_[1]);
+
+            }
             switch (userInput) {
                 case "bye":
                     output = "Bye. Hope to see you again soon!";
                     System.out.println(output);
                     break;
                 case "list":
                     int i = 1;
```

Comment:
since 'i' is the number of entries maybe you can create a variable name "numTask" . If you are using the standard forloop the it is fine to use 'i'. Standard forloop i mean (int i; i<a; i++)